docs/index.rst

```diff
@@ -28,6 +28,7 @@ This technical communication document, is intended to give assistance to develop
 .. toctree::
    :maxdepth: 2
 
+   gateways/index
```

Comment:
There's no `gateways` documentation folder neither index file. Why did you add it to the doctree?